### PR TITLE
Fix snapshot test regeneration job

### PR DESCRIPTION
**Pull request by PavelHolec**

### CODE REVIEW

File: Sources/Orbit/Support/BindingSource.swift [hunk 1]
```diff
@@ -6,6 +6,8 @@ public enum OptionalBindingSource<Value> {
     case state(Value)
 }
 
+extension Binding: Sendable where Value: Sendable {}
```
Comment (sjavora) [inline Sources/Orbit/Support/BindingSource.swift:9]: Why is this necessary? `Binding` already conforms to `Sendable`. Is this for some older OSes?

File: Sources/Orbit/Support/Previews/PreviewWrapper.swift [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import SwiftUI
 
 /// Registers Orbit fonts for preview content.
-public struct OrbitPreviewWrapper<Content: View>: View {
+@MainActor public struct OrbitPreviewWrapper<Content: View>: View {
```
Comment (sjavora) [inline Sources/Orbit/Support/Previews/PreviewWrapper.swift:4]: `View` is already `@MainActor` 🤔

File: Sources/Orbit/Support/BindingSource.swift (hunk 1 above)
Comment (PavelHolec) [reply Sources/Orbit/Support/BindingSource.swift:9]: Good point. This plus below were errors in my previous attempt to fix snapshot testing builds. It may not be necessary anymore, will check...

Comment (PavelHolec) [reply Sources/Orbit/Support/BindingSource.swift]: Removed

File: Sources/Orbit/Support/Previews/PreviewWrapper.swift (hunk 2 above)
Comment (PavelHolec) [reply Sources/Orbit/Support/Previews/PreviewWrapper.swift]: Removed

**Review by sjavora — APPROVED** [review]
